### ISSUE TITLE
Similar projects to be aware of

### ISSUE THREAD

**Issue by patcon** [twinbit/Drushistrano#1]
Hey there! just a heads up:
previousnext/capistrano-drupal#5

**Comment by twinbit** [comment]
@patcon Thanks for this heads up, we are ready to get involved on joint forces.

**Comment by patcon** [comment]
thanks dude! kinda slow going, and i probably got a little too wordy in the other thread, but think i'm going to make a doc page on d.o to summary all the different cap libraries, and summarize features and approaches.

hopefully that'll be a start to figuring out a sweet catch-all solution that'll work for most of us :)

**Comment by bryankennedy** [comment]
@patcon, did you ever make that doc page on d.o? I went looking but couldn't track it down.

**Comment by paolomainardi** [comment]
Hi guys, any progress on this ?

**Comment by bryankennedy** [comment]
I moved on to using Fabric myself.

**Comment by patcon** [comment]
No sorry folks, I spend most of my time on github rather than d.o lately.

Heads up on this project, btw: www.kraftwagen.org :)
